.circleci/unittest/linux/scripts/install.sh

```diff
@@ -7,11 +7,27 @@ unset PYTORCH_VERSION
 
 set -e
 
+case "$(uname -s)" in
+    Darwin*) os=MacOSX;;
+    *) os=Linux
+esac
+
 eval "$(./conda/bin/conda shell.bash hook)"
 conda activate ./env
 
 printf "* Installing PyTorch\n"
-conda install -y -c "pytorch-${UPLOAD_CHANNEL}" ${CONDA_CHANNEL_FLAGS} pytorch cpuonly
+(
+    if [ "${os}" == MacOSX ] ; then
+      # TODO: this can be removed as soon as linking issue could be resolved
+      #  see https://github.com/pytorch/pytorch/issues/62424 from details
+      MKL_CONSTRAINT='mkl==2021.2.0'
+    else
+      MKL_CONSTRAINT=''
+    fi
+    set -x
+    conda install -y -c "pytorch-${UPLOAD_CHANNEL}" "${CONDA_CHANNEL_FLAGS}" "$MKL_CONSTRAINT" pytorch cpuonly
```

Comment:
`CONDA_CHANNEL_FLAGS` and `MKL_CONSTANT` can be empty based on platforms, so the use of double-quotations here cause problem. The `command` will see it as option of empty string and it causes weird behavior.
```suggestion
    conda install -y -c "pytorch-${UPLOAD_CHANNEL}" ${CONDA_CHANNEL_FLAGS} ${MKL_CONSTRAINT} pytorch cpuonly
```

Reply:
BTW why does it have `cpuonly`? It does not hurt but it's not mandatory, is it? TorchText should build fine with non-CPU version of PyTorch (at least CUDA).

Reply:
> CONDA_CHANNEL_FLAGS and MKL_CONSTANT can be empty based on platforms, so the use of double-quotations here cause problem. The command will see it as option of empty string and it causes weird behavior.

Gotcha will change this back to remove the double quotes

> BTW why does it have cpuonly? It does not hurt but it's not mandatory, is it? TorchText should build fine with non-CPU version of PyTorch (at least CUDA).

Not sure what the historical context was for this. But also torchtext doesn't need the CUDA version of PyTorch to run. So does it make sense to install the CUDA version when it's not necessary for our library? In addition the CUDA version of PyTorch is a larger than the CPU only version.